package.json

```diff
@@ -25,7 +25,7 @@
   },
   "homepage": "https://github.com/zambezi/caballo-vivo#readme",
   "peerDependencies": {
-    "react": "^17.0.0"
+    "react": "^16.8.0||^17.0.0"
```

Comment:
Yeah, I guess this will need to be updated again when React 18 is out. Another alternative is `>=16.8.0`. Since we use react only [here](https://github.com/zambezi/caballo-vivo/blob/master/src/state-context.js#L1), using `>=` will work until React breaks context, which might be in the far future. I agree `"^16.8.0||^17.0.0"` is more conservative (and technically safer, if anyone uses a very old version of caballo-vivo with a future version of React), but it will have to be updated next year.